ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/ScanConfigs/Wizard/ScanConfigWizardForm.tsx

```diff
@@ -73,24 +74,51 @@ function ScanConfigWizardForm({ initialFormValues }: ScanConfigWizardFormProps):
         }
     }
 
+    function wizardStepChanged() {
+        handleProfilesUpdate();
+        setCreateScanConfigError('');
+    }
+
+    function scrollToAlert() {
+        if (alertRef.current) {
+            alertRef.current.scrollIntoView({
+                behavior: 'smooth',
+                block: 'start',
+            });
```

Comment:
In the few other places we use `scrollIntoView`, we only set behavior, https://github.com/search?q=repo%3Astackrox%2Fstackrox%20scrollIntoView&type=code

Would it be good to use this same pattern whenever we do a programmatic scroll, and add `block: 'start'` to all those instances, too?